src/averageAges.js

```diff
@@ -55,7 +62,18 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const women = people.filter(person =>
+    person.sex === 'f' && people.some(child => child.mother === person.name));
```

Comment:
If it's a mother, it's a woman too )

```suggestion
  const women = people.filter(person =>
    people.some(child => child.mother === person.name));
```